geomstats/geometry/pre_shape.py

```diff
@@ -378,6 +379,339 @@ def integrability_tensor(self, tangent_vec_a, tangent_vec_b, base_point):
+        :math: `A_Y A_X Y` and :math: `A_X Y` are computed here for the
+        Kendall shape space in the special case of quotient-parallel vector
+        fields :math: `X, Y` extending the values tg_vec_x and tg_vec_y by
+        parallel transport in a neighborhood. Such vector fields verify
+        :math: `\nabla^S_X^X = A_X X` and :math: `\nabla^S_X^Y = A_X Y`.
+
+        Parameters
+        ----------
+        tg_vec_x : array-like, shape=[..., {ambient_dim, [n, n]}]
+            Tangent vector at `base_point`.
+        tg_vec_y : array-like, shape=[..., {ambient_dim, [n, n]}]
+            Tangent vector at `base_point`.
+        base_point : array-like, shape=[..., {ambient_dim, [n, n]}]
+            Point of the total space.
+
+        Returns
+        -------
+        nabla_x_a_y_a_x_y : array-like, shape=[..., {ambient_dim, [n, n]}]
+            Tangent vector at `base_point`, result of :math:
+            `\nabla_X^S (A_Y A_X Y)` with `X = tg_vec_x` and `Y = tg_vec_y`.
+        a_x_a_y_a_x_y : array-like, shape=[..., {ambient_dim, [n, n]}]
+            Tangent vector at `base_point`, result of :math:
+            `A_X A_Y A_X Y` with `X = tg_vec_x` and `Y = tg_vec_y`.
+        nabla_x_a_x_y : array-like, shape=[..., {ambient_dim, [n, n]}]
+            Tangent vector at `base_point`, result of :math:
+            `\nabla_X^S (A_X Y)` with `X = tg_vec_x` and `Y = tg_vec_y`.
+        a_y_a_x_y : array-like, shape=[..., {ambient_dim, [n, n]}]
+            Tangent vector at `base_point`, result of :math:
+            `A_Y A_X Y` with `X = tg_vec_x` and `Y = tg_vec_y`.
+        a_x_y : array-like, shape=[..., {ambient_dim, [n, n]}]
+            Tangent vector at `base_point`, result of :math:
+            `A_X Y` with `X = tg_vec_x` and `Y = tg_vec_y`.
+
+        References
+        ----------
+        X Pennec. To be published.
+        """
+        # Vectors X and Y have to be horizontal.
+        if not gs.all(self.is_centered(base_point)):
+            raise ValueError('The base_point does not belong to the pre-shape'
+                             ' space')
+        if not gs.all(self.is_horizontal(tg_vec_x, base_point)):
+            raise ValueError('Tangent vector x is not horizontal')
+        if not gs.all(self.is_horizontal(tg_vec_y, base_point)):
+            raise ValueError('Tangent vector y is not horizontal')
+        # hor_x = self.horizontal_projection(tangent_vec_x, base_point)
+        # hor_y = self.horizontal_projection(tangent_vec_y, base_point)
+
+        p_top = Matrices.transpose(base_point)
+        p_top_p = gs.matmul(p_top, base_point)
+
+        def sylv_p(mat_b):
+            return gs.linalg.solve_sylvester(
+                p_top_p, p_top_p, mat_b - Matrices.transpose(mat_b))
+
+        y_top = Matrices.transpose(tg_vec_y)
+        x_top = Matrices.transpose(tg_vec_x)
+        x_y_top = gs.matmul(y_top, tg_vec_x)
+        omega_xy = sylv_p(x_y_top)
```

Comment:
This is possible. This adds four letters to the 53 occurrences in the file (sometimes 3 times on one line, which makes it go multi-line...). I tried to see what it gives, but I cannot make the link anymore with the latex formulas. In my opinion, this is deeply obscuring the code without bringing useful information. 

Since Omega was used as a notation for a skew symmetric matrix, would you prefer `skew_xy`? I personally don't find it clearer, but it is at least shorter.